### ISSUE TITLE
[Errors] Automatic Semicolon Insertion in Classes

### ISSUE THREAD

**Issue by mhegazy** [microsoft/TypeScript#126]
``` TypeScript
class A {
    var
    x = 10;
}
```

Expected: No parse error; ASI implies that 'var' is parsed as a separate property.

Actual: Parser errors with "Unexpected token. A constructor, method, accessor, or property was expected."
